docs/source/model_definition.md

```diff
@@ -90,13 +95,8 @@ head = Head(
 model = Model(head)
 ```
 
-### Tying embeddings
-For `NextItemPredictionTask` we have added a best practice, **Tying Embeddings**, proposed originally by the NLP community to tie the weights of the input (item id) embedding matrix with the output projection layer. Not only do tied embeddings reduce the memory requirements significantly, but our own experimentation during [recent competitions](https://resources.nvidia.com/en-us-merlin/recommendation-syste?lx=97GH0Q) and empirical analysis detailed in our [paper](https://dl.acm.org/doi/10.1145/3460231.3474255) and [online appendix](https://github.com/NVIDIA-Merlin/publications/blob/main/2021_acm_recsys_transformers4rec/Appendices/Appendix_A-Techniques_used_in_Transformers4Rec_Meta-Architecture.md) show how effective this method is.  It is enabled by default, but can be disabled by setting weight_tying=False).
-
+### Tying Embeddings
+For `NextItemPredictionTask`, we recommend **Tying Embeddings**, which was initially proposed by the NLP community to tie the weights of the input (item id) embedding matrix with the output projection layer. Not only do tied embeddings reduce the memory requirements significantly, but our own experimentation during [recent competitions](https://resources.nvidia.com/en-us-merlin/recommendation-syste?lx=97GH0Q) and empirical analysis detailed in our [Transformers4Rec: Bridging the Gap between NLP and Sequential / Session-Based Recommendation paper](https://dl.acm.org/doi/10.1145/3460231.3474255) and [online paper appendix](https://github.com/NVIDIA-Merlin/publications/blob/main/2021_acm_recsys_transformers4rec/Appendices/Appendix_A-Techniques_used_in_Transformers4Rec_Meta-Architecture.md) demonstrate that this method is very effective. It is enabled by default, but can be disabled by setting `weight_tying` to `False`.
 
 ## Regularization
-
-The library supports a number of regularization techniques like Dropout, Weight Decay, Softmax Temperature Scaling, Stochastic Shared Embeddings, and Label Smoothing. In our extensive experimentation hypertuning all regularization techniques for different dataset we found that the Label Smoothing was particularly useful at improving both train and validation accuracy and better calibrating the predictions.
-
-
-More details of the options available for each building block can be found in our **API Documentation**.
+Transformers4Rec supports a number of regularization techniques such as Dropout, Weight Decay, Softmax Temperature Scaling, Stochastic Shared Embeddings, and Label Smoothing. In our extensive experimentation, we hypertuned all regularization techniques for different datasets and found out that the Label Smoothing was particularly useful at improving both train and validation accuracy and better at calibrating the predictions.
```

Comment:
nit: Unless someone tells me otherwise, I think that the techniques like "dropout" should be lower case. I can't think of a reason why they would be proper nouns.